### ISSUE TITLE
Cannot reach the reported mAP using the pre-trained DSFD

### ISSUE THREAD

**Issue by JaywongWang** [Ir1d/DARKFACE_eval_tools#4]
Hi,
I tested DarkFace images using code from https://github.com/yxlijun/DSFD.pytorch (VGG backbone), but only got  mAP of 32.8%. However, your paper reports an mAP of 51.6%.  I am not sure why. Could you provide more testing details (such as input image size, NMS threshold, etc.)?

**Comment by Ir1d** [comment]
Hi, if I got it correct, the map of DSFD on test set should be 48%, the paper will be updated soon.
An example of DSFD can be found at https://github.com/Ir1d/docker_sample/tree/master/submission/DSFD .
Hope this helps.
